api_views/test_func.py

```diff
@@ -7,6 +7,13 @@ def mock_bad_implementation(r: str, p: int, g: str):
     x = requests.get(a, headers=h)
 
 
-def fetch_data_test_v1(api_url):
-    response = requests.get(api_url)
-    return response.text
\ No newline at end of file
+def fetch_data(api_url):
```

Comment:
Consider adding timeout parameter to the requests.get() call to prevent hanging on slow responses.